src/app/shared/components/notifications/notifications.component.ts

```diff
@@ -61,14 +65,33 @@ export class NotificationsComponent implements OnInit, AfterViewChecked, OnDestr
       };
     });
 
+    this.actions$
+      .pipe(
+        ofActionSuccessful(ReadUsersNotificationsByType),
+        takeUntil(this.destroy$),
+        switchMap((payload) => {
+          if (payload.notificationType === NotificationType.Application) {
```

Comment:
What if there is another type of notification? 
Probably an error will be thrown because we don't return an Observable